suite/tests/CMakeLists.txt

```diff
@@ -5396,6 +5396,13 @@ if (UNIX)
   if (NOT APPLE)
     tobuild(linux.thread linux/thread.c)
     tobuild(linux.threadexit linux/threadexit.c)
+
+    if (X86 AND X64 AND DR_HOST_X86 AND DR_HOST_X64)
+      tobuild_ci(linux.threadset linux/threadset.c "" "" "")
```

Comment:
Should this have "flush" or "synchall" in the name?  I'm not sure what the "set" in "threadset" is?